desktop/cmp/rest/RestGridModel.js

```diff
@@ -55,6 +56,20 @@ export const deleteAction = {
     actionFn: ({record, gridModel}) => gridModel.restGridModel.confirmDeleteRecord(record)
 };
 
+export const bulkDeleteAction = {
+    text: 'Delete',
+    icon: Icon.delete(),
+    intent: 'danger',
+    recordsRequired: true,
+    actionFn: ({gridModel}) => gridModel.restGridModel.confirmBulkDeleteRecords()
+};
```

Comment:
Could use this everywhere we want delete...rather than keeping old single del around. Then bulk is driven by multi select enabled or not. Is this too dangerous?